### ISSUE TITLE
Problem installing freqtrade - Command "python setup.py egg_info" failed with error code 1 

### ISSUE THREAD

**Issue by andremog** [gcarq/freqtrade#55]
Hi, I am trying to set up a VPS to run some tests but did not manage to install the bot. I am using CentOS 6. I already searched for a solution online but did not manage to solve this by myself. Does anyone know how to solve this? Thanks

I followed the installation instructions but the following command

> pip3.6 install -e .

returns:

> Obtaining file:///XXXXX/freqtrade-develop
>     Complete output from command python setup.py egg_info:
>     Traceback (most recent call last):
>       File "<string>", line 1, in <module>
>       File "/XXXXXX/freqtrade-develop/setup.py", line 3, in <module>
>         from freqtrade import __version__
>       File "/XXXXX/freqtrade-develop/freqtrade/__init__.py", line 3, in <module>
>         from . import main
>       File "/XXXXX/freqtrade-develop/freqtrade/main.py", line 10, in <module>
>         from jsonschema import validate
>     ModuleNotFoundError: No module named 'jsonschema'
>     
>     ----------------------------------------
> Command "python setup.py egg_info" failed with error code 1 in /XXXXX/freqtrade-develop/
>

**Comment by EricLim110** [comment]
I am using Windows 10 but I came across the same problem. Not sure the solution would be the same for your OS but it is worth giving it a shot.

I had to first install NumPy and install a .whl version of SciPy manually. (Pretty sure it can be easily found through Google for your OS)

Then I removed SciPy from requirements.txt file and run pip install -r requirements.txt again. Take note  that all these is done in the virtual environment. Everything worked fine for me after all these.
